### PR TITLE
fix: remove inconsistencies and match structure

### PR DESCRIPTION
**Pull request by Tomgroot**

### CODE REVIEW

File: src/Message/Request/PurchaseRequest.php [hunk 1]
```diff
@@ -214,16 +258,6 @@ public function setServiceId($value)
...
     }
 
-    public function getCardToken()
-    {
-        return $this->getParameter('cardToken');
-    }
-
-    public function setCardToken($value)
-    {
-        return $this->setParameter('cardToken', $value);
```
Comment (Tomgroot) [inline src/Message/Request/PurchaseRequest.php:224]: This is not appended to the redirect url in the omnipay anymore. Just in our back end which is far more logical.

File: src/Message/Request/FetchIssuersRequest.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 /**
  * Class FetchIssuersRequest
- * @package Omnipay\Paynl\Message\Request
+ * @package Omnipay\PaynlV3\Message\Request
```
Comment (Grisgruis) [inline src/Message/Request/FetchIssuersRequest.php:9]: @Tomgroot did they just forget to adjust this? This is still from the fork right?

File: tests/Message/PurchaseRequestTest.php [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+        $this->request->setClientIp('10.0.0.5');
+        $this->request->setReturnUrl('https://www.pay.nl');
+
+        $response = $this->request->send();
+        $this->assertInstanceOf(PurchaseResponse::class, $response);
+
+        $this->assertFalse($response->isSuccessful());
+        $this->assertTrue($response->isRedirect());
+
+        $this->assertIsString($response->getTransactionReference());
```
Comment (Grisgruis) [inline tests/Message/PurchaseRequestTest.php:35]: Lets check actual value here as well, one of the most important identifiers

File: tests/Message/RefundRequestTest.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        $this->setMockHttpResponse('RefundRequestSuccess.txt');
+
+        $transactionId = '52200435050X2314';
+
+        $this->request->setTransactionReference($transactionId);
+
+        $response = $this->request->send();
+
+        $this->assertInstanceOf(RefundResponse::class, $response);
+        $this->assertEquals($transactionId, $response->getTransactionReference());
```
Comment (Grisgruis) [inline tests/Message/RefundRequestTest.php]: Iam wondering if the 'refund' has a new transactionReference (we have now through API)

File: tests/Mock/FetchTransactionRequestPaid.txt [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+Pragma: no-cache;
+X-Frame-Options: SAMEORIGIN;
+X-XSS-Protection: 1; mode=block;
+X-Content-Type-Options: nosniff;
+Connection: close;
+Content-Type: application/json;
+
+{
+	"id": "672dee93-5223-8f13-1f71-771807002314",
+	"serviceId": "SL-7718-0700",
```
Comment (Grisgruis) [inline tests/Mock/FetchTransactionRequestPaid.txt]: This is ours right? (because we proabaly PR-back into public repo, remove this one?

Comment (Grisgruis) [reply tests/Mock/FetchTransactionRequestPaid.txt]: yes it can be public true

File: tests/Message/RefundRequestTest.php (hunk 4 above)
Comment (Grisgruis) [reply tests/Message/RefundRequestTest.php]: this should be `EX-9167-1816-4111` in the testcase not 52200435050X2314

File: src/Message/Request/FetchIssuersRequest.php (hunk 2 above)
Comment (Tomgroot) [reply src/Message/Request/FetchIssuersRequest.php:9]: Yes they did forget. Namespace has been changed to `Omnipay\PaynlV3` by them (see top of this file), do we want to change this back to the old one? We also change the namespace in api.

File: tests/Mock/FetchTransactionRequestPaid.txt (hunk 5 above)
Comment (Tomgroot) [reply tests/Mock/FetchTransactionRequestPaid.txt]: will remove, indeed public but not neat to have it here

File: src/Message/Request/FetchIssuersRequest.php (hunk 2 above)
Comment (Grisgruis) [reply src/Message/Request/FetchIssuersRequest.php:9]: No it's fine, was just wondering what kind of job they did

**Pull request merged**